src/registry/utils.ts

```diff
@@ -79,12 +80,16 @@ export async function extractReadonly(
 }
 
 export async function wrapTransaction(
-  conn: ReadConfig,
+  conn: Config,
```

Comment:
Is this going to cause issues at all? Before we were only requiring a read config. I guess this is ok, because we likely won't be wrapping a transaction at all if we are just in readOnly mode, right?

Reply:
I _think_ it should be ok.  All the tests pass and in the little demo app @dtbuchholz setup it seemed to work.